src/components/dock/dock.scss

```diff
@@ -70,7 +70,7 @@ nav {
     flex-grow: 1;
 
     padding: var(--dock-padding); // needed for the focus effect
-
+    padding-bottom: calc((var(--dock-padding) * 2.4));
```

Comment:
This approach is logical. But it has a few problems.
1. It adds some extra "unnecessary" distance between the buttons. 
    <img width="107" alt="image" src="https://github.com/user-attachments/assets/7f280166-da84-4f1b-a455-0810b46f1790"> 
    It's not a a big deal. But in general, we try to **save** as much space as possible. So if we can achieve the same result without adding the extra space, it'd be absolutely desirable.
2. In the mobile layout (scroll further down in the documentations examples), there is also the same padding-bottom. But it is not needed there.
    <img width="346" alt="image" src="https://github.com/user-attachments/assets/c1b4ba9c-f64b-4fab-a860-98878493ff61"> So if we want to go with this approach, we need to make sure that the mobile layout is fine.